### PR TITLE
Remove Functionality

### PR DESCRIPTION
**Pull request by jroodman**
Updated README for the repo and added Remove functionality for items that a user has previously added.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,107 @@
-# Pantry
-A Ruby on Rails backend for an Alexa Skill. 
+# Pantry Keeper
+
+A custom Amazon Alexa skill powered by a custom Ruby on Rails application. Pantry Keeper provides a modern, state-of-the-art solution to keeping track of grocery items and how long they will last. With ongoing development, Pantry Keeper has the potential to more new and innovative services to its end-users in the realms of food safety, food storage, and cooking.
```
Comment (rysospicy) [inline README.md]: Probably intended to say something other than "potential to more new" in that last sentence.

File: app/lib/handlers/transaction_intent_request_handler.rb [hunk 2]
```diff
@@ -51,13 +51,37 @@ def add_items
...
+          item.quantity = (item.quantity - v >= 0) ? item.quantity - v : 0
+          if item.quantity == 0
+            item.destroy
+            item = Item.owned_by(user_id).where(name: k).order(:created_at).first
+          end
+        end
+        item_meta[:quantity] -= v if v.positive?
+        item.save if item.present?
+        item_meta
+      end
```
Comment (rysospicy) [inline app/lib/handlers/transaction_intent_request_handler.rb:90]: This seems a lot to have packed into a single method.  I'd refactor this if possible

**Review by rysospicy — APPROVED** [review]
👍

File: README.md (hunk 1 above)
Comment (jroodman) [reply README.md]: Maybe.. Maybe not...

**Pull request merged**